causallearn/search/HiddenCausal/GIN/FisherTest.py

```diff
@@ -0,0 +1,45 @@
+#-------------------------------------------------------------------------------
+# Name:        模块1
+# Purpose:
+#
+# Author:      YY
+#
+# Created:     03/03/2021
+# Copyright:   (c) YY 2021
+# Licence:     <your licence>
+#-------------------------------------------------------------------------------

```

Comment:
We should delete this, or fill it.

In the open-source code, we should maintain high code quality --- we shouldn't allow any unnecessary comments in the code. 

Sorry there are lots of rules in industry code that are different from research code :(